src/App.tsx

```diff
@@ -1,156 +1,103 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { ErrNotification } from './Component/ErrNotification/ErrNotification';
+import { Footer } from './Component/Footer/Footer';
+import { Header } from './Component/Header/Header';
+import { TodoList } from './Component/TodoList/TodoList';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { ErrorType } from './types/ErrorType';
+import { FilterBy } from './types/FilterBy';
 
-const USER_ID = 0;
+const USER_ID = 11091;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  const [activeTodosCount, setActiveTodosCount] = useState(0);
+  const [complitedTodosCount, setComplitedTodosCount] = useState(0);
+
+  const [filterValue, setFilterValue] = useState(FilterBy.ALL);
+
+  const [errorMessage, setErrorMessage] = useState<ErrorType | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isHidden, setIsHidden] = useState(false);
+
+  const loadTodos = async () => {
+    setErrorMessage(null);
+    setIsHidden(false);
+    setIsLoading(true);
+    try {
+      const loadedTodos = await getTodos(USER_ID);
+
+      setTodos(loadedTodos);
+    } catch {
+      setErrorMessage(ErrorType.LOAD);
+      setTimeout(() => {
+        setIsHidden(true);
+      }, 3000);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    loadTodos();
+  }, []);
+
+  useEffect(() => {
+    const activeTodos = todos.filter(({ completed }) => !completed);
+    const complitedTodos = todos.filter(({ completed }) => completed);
```

Comment:
can't figure out how to avoid this step. We need to have a count of active and completed todos and use them in other components. Like in filter component we have count of active todos. Even if im filtering by completed status i need to know count of active todos.